keras_cv/src/models/stable_diffusion_v3/cfg_denoiser.py

```diff
@@ -0,0 +1,9 @@
+from keras import ops
```

Comment:
Since KerasCV still has support for Keras2, we will need to update the import from
from keras import ops -> from keras_cv.src.backend import ops